mockgen/model/model_test.go

```diff
@@ -0,0 +1,32 @@
+package model
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestImpPath(t *testing.T) {
+	nonVendor := "github.com/foo/bar"
+	if nonVendor != impPath(nonVendor) {
+		t.Errorf("")
+
+	}
+	testCases := []struct {
+		input string
+		want  string
+	}{
+		{"foo/bar", "foo/bar"},
+		{"vendor/foo/bar", "foo/bar"},
+		{"/vendor/foo/bar", "foo/bar"},
+		{"qux/vendor/foo/bar", "foo/bar"},
+		{"qux/vendor/foo/vendor/bar", "bar"},
```

Comment:
Add some test cases for the bug we found in the last review, where the word vendor appears as part of the directory:
"govendor/foo"
"foo/govendor/bar"
"vendors/foo"
"foo/vendors/bar"

Reply:
done. also added one for `vendor/foo/vendor/bar`